proposals/lip-0049.md

```diff
@@ -518,6 +523,7 @@ registrationCCMParamsSchema = {
 * `name`: name registered on the mainchain.
 * `chainID`: chain ID registered on the mainchain.
 * `messageFeeTokenID`: ID of the token used for cross-chain message fees.
+* `minReturnFeePerByte`: the minimum value of the CCM fee required for the CCM to be bounced in case of execution errors.
```

Comment:
```suggestion
* `minReturnFeePerByte`: the minimum value of the CCM fee per byte required for the CCM to be bounced in case of execution errors.
```